docs/bundler/migration.md

```diff
@@ -0,0 +1,1107 @@
+  Bun.build({
+    entrypoints: ["./index.tsx"],
+    naming: {
+      chunk: "[name].[ext]",
+    },
+  });
+  ```
+
+---
+
+- `color`
+- n/a
+- Always `true`
+
+---
+
+- `conditions`
+- n/a
+- Not supported. Export conditions priority is determined by `target`.
+
+---
+
+- `define`
+- `define`
+- Not supported in JS API
+
+---
+
+- `drop`
+- n/a
+- Not supported
+
+---
+
+- `entryNames`
+- `naming` or `naming.entry`
+- Bun supports a `naming` key that can either be a string or an object. Uses same templating syntax as esbuild, but `[ext]` must be included explicitly.
+
+  ```ts
+  Bun.build({
+    entrypoints: ["./index.tsx"],
+    // when string, this is equivalent to entryNames
+    naming: "[name].[ext]",
+
+    // granular naming options
+    naming: {
+      entry: "[name].[ext]",
+      asset: "[name].[ext]",
+      chunk: "[name].[ext]",
+    },
+  });
+  ```
+
+---
+
+- `entryPoints`
+- `entrypoints`
+- Capitalization difference
+
+---
+
+- `external`
+- `external`
+- No differences
+
+---
+
+- `footer`
+- n/a
+- Not supported
+
+---
+
+- `format`
+- `format`
+- Only supports `"esm"` currently. Support for `"cjs"` and `"iife"` is planned.
+
+---
+
+- `globalName`
+- n/a
+- Not supported
+
+---
+
+- `ignoreAnnotations`
+- n/a
+- Not supported
+
+---
+
+- `inject`
+- n/a
+- Not supported
+
+---
+
+- `jsx`
+- `jsx`
+- Not supported in JS API, configure in `tsconfig.json`
+
+---
+
+- `jsxDev`
+- `jsxDev`
+- Not supported in JS API, configure in `tsconfig.json`
+
+---
+
+- `jsxFactory`
+- `jsxFactory`
+- Not supported in JS API, configure in `tsconfig.json`
+
+---
+
+- `jsxFragment`
+- `jsxFragment`
+- Not supported in JS API, configure in `tsconfig.json`
+
+---
+
+- `jsxImportSource`
+- `jsxImportSource`
+- Not supported in JS API, configure in `tsconfig.json`
+
+---
+
+- `jsxSideEffects`
+- `jsxSideEffects`
+- Not supported in JS API, configure in `tsconfig.json`
+
+---
+
+- `keepNames`
+- n/a
+- Not supported
+
+---
+
+- `legalComments`
+- n/a
+- Not supported
+
+---
+
+- `loader`
+- n/a
+- Not supported in JS API
+
+---
+
+- `logLevel`
+- n/a
+- Not supported
+
+---
+
+- `logLimit`
+- n/a
+- Not supported
+
+---
+
+- `logOverride`
+- n/a
+- Not supported
+
+---
+
+- `mainFields`
+- n/a
+- Not supported
+
+---
+
+- `mangleCache`
+- n/a
+- Not supported
+
+---
+
+- `mangleProps`
+- n/a
+- Not supported
+
+---
+
+- `mangleQuoted`
+- n/a
+- Not supported
+
+---
+
+- `metafile`
+- `manifest`
+- When `manifest` is `true`, the result of `Bun.build()` will contain a `manifest` property. The manifest is compatible with esbuild's metafile format.
+
+---
+
+- `minify`
+- `minify`
+- In Bun, `minify` can be a boolean or an object.
+
+  ```ts
+  Bun.build({
+    entrypoints: ['./index.tsx'],
+    // enable all minification
+    minify: true
+
+    // granular options
+    minify: {
+      identifiers: true,
+      syntax: true,
+      whitespace: true
+    }
+  })
+  ```
+
+---
+
+- `minifyIdentifiers`
+- `minify.identifiers`
+- Nested object syntax
```

Comment:
up to you but what if it just said "see above" instead of "nested object syntax"